Error in git repo regex when inserting equations

<!--lint disable first-heading-level-->

## Checklist

> Please ensure the following tasks are completed before filing an issue.

-   [x] Read and understood the [Code of Conduct][code-of-conduct].
-   [x] Searched for existing issues and pull requests.
-   [x] If this is a general question, searched the [FAQ][faq] for an existing answer.
-   [x] If this is a feature request, the issue name begins with `RFC:`.

## Description

> Description of the issue (or feature request).

Encountered error when running:

```bash
$ make MARKDOWN_FILTER=.*/math/base/special/fresnel/.* markdown-img-equations-src-urls
out: git@github.com:stdlib-js/stdlib.git

lib/node_modules/@stdlib/math/base/special/fresnel/README.md
  1:1  error  Error: Cannot parse given file `etc/remark/.remarkrc.js`
Error: Cannot parse script `tools/remark/plugins/remark-img-equations-src-urls/lib/index.js`
TypeError: Cannot read property '1' of null
    at git (/Users/rreusser/projects/node/stdlib/stdlib/tools/remark/plugins/remark-img-equations-src-urls/lib/git.js:50:28)
    at Object.<anonymous> (/Users/rreusser/projects/node/stdlib/stdlib/tools/remark/plugins/remark-img-equations-src-urls/lib/git.js:75:18)
```

The remote is parsed is parsed to expect `github.com/stdlib/stdlib.git` instead of also allowing `github.com:stdlib/stdlib.git`. Actually getting the rawgit url correct is a slightly different matter, but it seems like it'd be reasonable to parse it either way.

## Related Issues

> Does this issue (or feature request) have any related issues?

No.

## Questions

> Any questions for reviewers?

No.

<!-- <links> -->

[code-of-conduct]: https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md

[faq]: https://github.com/stdlib-js/stdlib/blob/develop/FAQ.md

<!-- </links> -->

--- a/tools/remark/plugins/remark-img-equations-src-urls/lib/git.js
+++ b/tools/remark/plugins/remark-img-equations-src-urls/lib/git.js
@@ -9,7 +9,7 @@ var exec = require( 'child_process' ).execSync;
 // VARIABLES //
 
 // Regular expression to extract a repository slug:
-var RE = /(?:.+github\.com\/)(.+)\.(?:.+)/;
+var RE = /(?:.+github\.com)(?:\/|:)(.+)\.(?:.+)/;
 
 
 // MAIN //

